core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -495,9 +495,10 @@ message RDDOperationGraphWrapper {
 }
 
 message StreamingQueryData {
-  string name = 1;
+  optional string name = 1;
+  // convert from uuid, id should not be null
```

Comment:
let's simply convert all the string fields. We can even have tests to check if there is any string which is not optional.

Reply:
OK